src/main/java/com/google/devtools/build/lib/rules/objc/AppleBinary.java

```diff
@@ -291,7 +291,7 @@ private static ExtraLinkArgs getRequiredLinkopts(RuleContext ruleContext)
     switch (binaryType) {
       case LOADABLE_BUNDLE:
         extraLinkArgs.add("-bundle");
-        extraLinkArgs.add("-Xlinker", "-rpath", "-Xlinker", "@loader_path/Frameworks");
+        extraLinkArgs.add("-Wl,-rpath,@loader_path/Frameworks");
```

Comment:
FYI I prepared an identical patch last night and sanity checked that it didn't break anything.  It should be landing shortly.

Reply:
sounds great, I'll revert once that is pushed out